lib/data_update_scripts/20211115154021_nullify_invalid_tag_fields.rb

```diff
@@ -0,0 +1,34 @@
+module DataUpdateScripts
+  class NullifyInvalidTagFields
+    def run
+      Tag.where(bg_color_hex: "").or(Tag.where(text_color_hex: "")).find_each do |tag|
+        # presence here selects non-empty strings if set  (avoiding unsetting valid values)
+        # but nullifies empty tag colors
+        tag.update_columns(
+          bg_color_hex: tag.bg_color_hex.presence,
+          text_color_hex: tag.text_color_hex.presence,
+        )
+      end
+
+      # normalize empty string to nil (alias_for? behaved correctly already), this is just cleanup
+      Tag.where(alias_for: "").update(alias_for: nil)
+
+      Tag.where.not(alias_for: nil).find_each do |aliased_tag|
```

Comment:
It's possible that this entire block (validate or nullify alias_for) can be removed - in practice it's not expected to be either long running or to find/fix anything, it _could_ and I was thinking about the case, but I'm happy to remove this update from the script.